apps/passport-server/src/services/generic-issuance/genericIssuanceService.ts

```diff
@@ -243,20 +294,43 @@ export async function startGenericIssuanceService(
   }
 
   const pkeyEnv = process.env.GENERIC_ISSUANCE_EDDSA_PRIVATE_KEY;
-
   if (pkeyEnv == null) {
     logger(
       "[INIT] missing environment variable GENERIC_ISSUANCE_EDDSA_PRIVATE_KEY"
     );
     return null;
   }
 
+  const projectIdEnv = process.env.STYTCH_RROJECT_ID;
+  if (projectIdEnv == null) {
+    logger("[INIT] missing environment variable STYTCH_RROJECT_ID");
+    return null;
+  }
+
+  const secretEnv = process.env.STYTCH_SECRET;
+  if (secretEnv == null) {
+    logger("[INIT] missing environment variable STYTCH_SECRET");
+    return null;
+  }
+
+  const stytchClient = new stytch.Client({
+    project_id: projectIdEnv,
+    secret: secretEnv
+  });
+
+  const genericIssuanceClientUrl = process.env.GENERIC_ISSUANCE_CLIENT_URL;
+  if (genericIssuanceClientUrl == null) {
+    throw new PCDHTTPError(500, "[INIT] missing GENERIC_ISSUANCE_CLIENT_URL");
```

Comment:
this shouldn't be a pcd http error as this isn't in the call stack of an express route handler